htdocs/takepos/takepos.php

```diff
@@ -89,6 +89,8 @@
 $subcategories = array();
 foreach($categories as $key => $categorycursor)
 {
+	if (empty($categorycursor['visible'])) continue;	// Hide not visible categories
```

Comment:
Currently there is no way to set if a category is visible or not. And by default all category are not visible.
So if we add this line, no categories are never selected on a default dolibarr install.
May be we should set the default value of "visible" property of category to 1 instead of 0 ? If we do that, we must also migrate the data to update value from 0 to 1.

Reply:
@eldy 
the goal is to hide certain categories in the POS, it allows to classify products only accessible in Dolibarr and not in the POS... you see ?